akka-docs-new/src/test/scala/tutorial_3/Device.scala

```diff
@@ -0,0 +1,44 @@
+/**
+ * Copyright (C) 2009-2017 Lightbend Inc. <http://www.lightbend.com>
+ */
+package tutorial_3
+
+import akka.actor.{ Actor, ActorLogging, Props }
+import tutorial_3.Device.{ ReadTemperature, RecordTemperature, RespondTemperature, TemperatureRecorded }
+import tutorial_3.DeviceManager.{ DeviceRegistered, RequestTrackDevice }
+
+object Device {
+
+  def props(groupId: String, deviceId: String): Props = Props(new Device(groupId, deviceId))
+
+  case class RecordTemperature(requestId: Long, value: Double)
+  case class TemperatureRecorded(requestId: Long)
+
+  case class ReadTemperature(requestId: Long)
+  case class RespondTemperature(requestId: Long, value: Option[Double])
```

Comment:
Timestamps are realistic, I just didn't want to complicate (they need to come from devices, you need to drop old ones, etc). I am fine with any names for the messages though.